test/assert-test.js

```diff
@@ -1649,4 +1649,45 @@ describe("assert", function () {
                           "    actual = { foo: 1 }");
         });
     });
+
+    if (typeof Symbol === "function") {
+        describe("with symbol method names", function () {
+            function setupSymbol(symbol) {
+                this.obj = {};
+                this.obj[symbol] = function () {};
+                sinonSpy(this.obj, symbol);
+            }
+
+            beforeEach(function () {
+                this.setupSymbol = setupSymbol.bind(this);
+
+                /*eslint consistent-return: "off"*/
+                this.message = function (method) {
+                    try { // eslint-disable-line no-restricted-syntax
+                        sinonAssert[method].apply(sinonAssert, [].slice.call(arguments, 1));
+                    } catch (e) {
+                        return e.message;
+                    }
+                };
+            });
+
+            it("assert.called exception message with symbol with description", function () {
```

Comment:
The sentence is not valid English. Make it read out aload in a natural way, for instance `it("should create an exception message for symbols that describe the symbol"`